guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -313,6 +317,7 @@ When you start the physical host and boot from the ISO or the USB storage device
 ifdef::satellite,orcharhino[]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
+endif::[]
```

Comment:
I think lines 206/210 and 313/317 needs to be simplified if the whole section is "non Satellite" only.